ports/renesas-ra/Makefile

```diff
@@ -403,6 +431,13 @@ HAL_SRC_C += $(addprefix ra/,\
 
 endif
 
+ifeq ($(BOARD),$(filter $(BOARD),VK_RA4W1 VK_RA6M3 VK_RA6M5))
```

Comment:
I haven't samples of other boards to test with, so I am not sure if it will work on them If it is released for the series. This only @TakeoTakahashi2020 can confirm. Ask him.

Reply:
Please change it.  I can check it on EK_RA4W1 and report if there are any issues.